cargo-dist/templates/installer/npm/run.js.j2

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env node
 
 const { run } = require("./binary");
+{%- if bin is defined %}
 run({{ bin }});
```

Comment:
Did we run into a scenario where `bin` is undefined? I think that would actually be a for-real error here, not something for us to work around.

Reply:
The code is doing something dirty here — it's rendering the whole `npm/` folder passing `NpmInstallerInfo` (which does _not_ have a `bin` property) — and then removing the generated `run.js` (which presumably has a null or whatever), and then re-generating from `run.js.j2` for every binary, see `fn mangle_run_js()` in `src/backend/installer/npm.rs`

Reply:
(What this PR does is not the proper fix, but I was, once upon a time, trying to keep this PR small)